WorkflowLayer/SpritzWorkflow.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using System.Diagnostics;
+
+namespace WorkflowLayer
+{
+    public enum MyWorkflow
```

Comment:
We should type check the SpritzWorkflow objects, rather than using an enum. Enums are more for when there's a defined and strictly limited set of options.